tensorflow/lite/micro/kernels/depthwise_conv_test.cc

```diff
@@ -983,4 +936,59 @@ TF_LITE_MICRO_TEST(Int8Input32x1Filter32x1ShouldMatchGolden) {
                               kQuantizationTolerance, kTensorsSize, tensors));
 }
 
+
+#if !defined(VISION_P6) && !defined(XTENSA)  
```

Comment:
should we only disable for P6 and leave for other XTENSA  targets?

Reply:
xtensa was already disabled

Reply:
yes, let's enable. I would think that removing the ifdef(XTENSA) was missed in the change when we enabled optimized kernels to also support int4.

Reply:
Done !